lib/std/Build.zig

```diff
@@ -1727,20 +1727,48 @@ pub fn fmt(b: *Build, comptime format: []const u8, args: anytype) []u8 {
     return std.fmt.allocPrint(b.allocator, format, args) catch @panic("OOM");
 }
 
+// Keep this logic in sync with std.process.Child.windowsCreateProcessSupportsExtension().
+fn supportedWindowsProgramExtension(ext: [] const u8) bool {
+    inline for (&.{ "bat", "cmd", "com", "exe" }) |e| {
```

Comment:
These should include the `.`

```suggestion
    inline for (&.{ ".bat", ".cmd", ".com", ".exe" }) |e| {
```